src/workflows-app/components/LeftNavigationPanel.ts

```diff
@@ -0,0 +1,156 @@
+import _ from 'lodash/fp';
+import { useState } from 'react';
+import { div, h, span } from 'react-hyperscript-helpers';
+import Collapse from 'src/components/Collapse';
+import { Clickable } from 'src/components/common';
+import { centeredSpinner, icon } from 'src/components/icons';
+import colors from 'src/libs/colors';
+import { getConfig } from 'src/libs/config';
+import * as Style from 'src/libs/style';
+import HelpfulLinksBox from 'src/workflows-app/components/HelpfulLinksBox';
+import { SubmissionHistory } from 'src/workflows-app/SubmissionHistory';
+
+const subHeadersMap = {
+  'workspace-workflows': 'Workflows in this workspace',
+  'submission-history': 'Submission history',
+} as const;
+
+const findAndAddSubheadersMap = {
+  'featured-workflows': 'Featured workflows', // Update in follow-up tickets
+  'import-workflow': 'Import a workflow', // Update in follow-up tickets
+};
+
+export const LeftNavigationPanel = ({ name, namespace, workspace, loading }) => {
+  const [selectedSubHeader, setSelectedSubHeader] = useState<string>('workspace-workflows');
+
+  const isSubHeaderActive = (subHeader) => selectedSubHeader === subHeader;
+
+  const ListItem = ({ title, ...props }) =>
+    h(
+      Clickable,
+      {
+        style: {
+          display: 'flex',
+          alignItems: 'center',
+          flex: 'none',
+          width: '100%',
+          height: 50,
+          padding: '0 1.5rem',
+          color: colors.accent(1.1),
+          ...props.style,
+        },
+      },
+      [div({ style: { fontSize: 15, wordBreak: 'break-all' }, role: 'list' }, [title])]
+    );
+
+  return div({ role: 'main', style: { display: 'flex', flex: 1, height: 'calc(100% - 66px)' } }, [
+    div({ style: { minWidth: 330, maxWidth: 330, boxShadow: '0 2px 5px 0 rgba(0,0,0,0.25)', overflowY: 'auto' } }, [
+      _.map(([subHeaderKey, subHeaderName]) => {
+        const isActive = isSubHeaderActive(subHeaderKey);
+        return loading
+          ? centeredSpinner()
+          : h(
+              Clickable,
+              {
+                'aria-label': `${subHeaderKey}-header-button`,
+                style: { color: isActive ? colors.accent(1.1) : colors.accent(), fontSize: 16 },
```

Comment:
I'm not noticing a difference in colors/style when clicking on different subheaders. Could we also maybe bold the subheader when it's active?

Reply:
The mockups also show that the top-level sub-header that is selected also has a different background which should also be incorporated here.
![Screenshot 2023-09-11 at 1 21 12 PM](https://github.com/DataBiosphere/terra-ui/assets/16748522/f8750c5f-fd79-4d24-a242-13a79bedbd70)